src/main/java/io/fabric8/jenkins/openshiftsync/ConfigMapWatcher.java

```diff
@@ -190,6 +196,46 @@ private boolean containsSlave(ConfigMap configMap) {
                 podTemplate = xStream2.fromXML(entry.getValue());
 
                 if( podTemplate instanceof PodTemplate ) {
+                    PodTemplate pt = (PodTemplate)podTemplate;
+                    
+                    // if the image field is and imagestream related ref, without the internal docker registry, prepend it
+                    String image = pt.getImage();
+                    try {
+                        if (!OpenShiftUtils.hasRegistry(image)) {
```

Comment:
an image name like "openshift/foo:bar" is both a valid docker image registry reference (docker.io is implied) and a valid imagestreamtag (the bar tag of the foo imagestream in the openshift namespace).

I think any attempt to infer intent(imagestream[tag] ref vs dockerimage[tag] ref) from a single field is going to be fraught with peril.

i also think if we're going to do this, we should mandate that the value be an imagestream:tag, not just an imagestream.  Being pointed to just an imagestream is rarely sufficient to make the right decision about what the user wanted to get out of that imagestream.